content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -124,6 +124,16 @@ The Account Resource supports proprietary codes for:
 * Account.state codes are maintained in [Code Set 24451 Benefit Order Status](#code-set-24451-benefit-order-status) and [Code Set 4002640 Correspondence Status](#code-set-4002640-correspondence-status)
 * Account.identifier codes are maintained in [Code Set 28200 PFT Bill Alias Type](#code-set-28200-pft-bill-alias-type)
 
+<h4>ChargeItem</h4>
+
+The ChargeItem Resource supports proprietary codes for:
+
+* ChargeItem.code codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.reason codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.procedure codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.modifier codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
+* ChargeItem.revenue-code codes are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
```

Comment:
Should this be `ChargeItem.bill-code-schedule`? @alexbeckner1 Thoughts?

Reply:
I think I like "The Bill Code Schedule extensions on ChargeItem are maintained in"

Reply:
updated here to match similar to other resources. Let me know your thoughts if it is not accurate. 
https://github.com/cerner/fhir.cerner.com/pull/629/commits/331b6bd1e7c58662bb861c7068462fce4a455244#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R131

Reply:
Can we please use my exact verbiage from my previous comment. I feel like representing it as ChargeItem.bill-code-schedule is misleading as the resource path isn't accurate. Can we also update the Account.balance to "The Balance extensions on Account are maintained in Code Set [Code Set 18934 Currency Type](#code-set-18934-currency-type)" and update Account.state to "The State extensions on Account are maintained in Code Set [Code Set 24451 Benefit Order Status](#code-set-24451-benefit-order-status) and [Code Set 4002640 Correspondence Status](#code-set-4002640-correspondence-status)" This would put these extensions in line with the Custom Attribute Extension under http://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#encounters

Reply:
updated here in below commits
https://github.com/cerner/fhir.cerner.com/pull/629/commits/aaf9b89123fa1d765a6b012d9d9750b701b95ff3
https://github.com/cerner/fhir.cerner.com/pull/629/commits/a81d1be9fe21f0430082839ad380a84c52c83f81

Reply:
+1 to changes